tests/test_normalizers.py

```diff
@@ -9,15 +9,18 @@ def get_normed(normalizer, input_path):
     normed = normalizer.run(input_path)
     normed = normed.apply(lambda x: x.str.strip() if isinstance(x, str) else x).replace('', np.nan)
     return normed
- 
+
+def get_golden_file(path):
+    return pd.read_csv(path, sep='\t', skiprows=1)
```

Comment:
Instead of `skiprows=1`, it's better to do `comment='#'`

Reply:
I just tried this right now, and I realized that a few ARG annotation tools use '#' as part of their column headers. For example here: https://github.com/BigDataBiology/argNorm/blob/main/outputs/raw/abricate.argannot.tsv, the file column header is given as '#FILE'. So would it be better to stick with `skiprows=1` as I know for sure that the argNorm version will only be on the first line?

Reply:
I see.

Just make sure this is well documented. In particular, in the release notes, there needs to be a bold warning type thing of "We are changing the output format and all your scripts will break!"

Reply:
Added this in #70